style.css

```diff
@@ -617,6 +617,105 @@ textarea::placeholder {
   font-size: 20px;
 }
 
+/* --- Popup Section --- */
+.overlay {
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.6);
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  display: none;
+  backdrop-filter: blur(10px);
+}
+
+.popup-contain {
+  width: 50%;
+  margin: 0 auto;
+  padding: 24px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  z-index: 2;
+  background-color: #fff;
+  box-sizing: border-box;
+  border: 1px solid red;
+}
```

Comment:
Great job on implementing the changes as specified by the previous reviewer! 👏 However, there is a minor UI issue 🐞 that we need to address as part of the changes suggested by the previous reviewer: 
- [ ] On mobile screens, the content is squashed together, and due to that the contents are overflowing also there is no spacing and margins like that of the `Figma` design. To fix this issue kindly consider below suggestions:
   - [ ]  Increase the set `width` on mobile screens.
   - [ ] Add some spacing between your buttons.
   - [ ] Reduce the height of the pop-up image.
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td><img width="346" src="https://user-images.githubusercontent.com/107632780/247645032-7e9da6c2-9ed8-4f01-9477-33796e7f71f0.png" /></td>
<td>
<img width="372" alt="Screenshot 2023-06-21 at 21 38 16" src="https://github.com/Mahdi-Niazi/Portfolio_Mobile_Setup/assets/107192076/fbc56cdf-98b4-435b-911f-1c765d9116bd">
</td>
</tr>
</table>